src/java.base/share/classes/java/lang/runtime/SwitchBootstraps.java

```diff
@@ -136,42 +145,52 @@ private SwitchBootstraps() {}
      *
      * @throws NullPointerException if any argument is {@code null}
      * @throws IllegalArgumentException if any element in the labels array is null, if the
-     * invocation type is not not a method type of first parameter of a reference type,
+     * invocation type is not a method type of first parameter of a reference type,
```

Comment:
One thing we did often in the FFM API was to break up long `@throws` and split them into two or three `@throws` (javadoc allows the same exception to appear multiple times). Not saying you have to change this, just a possible suggestion.